README.md

```diff
@@ -330,6 +331,31 @@ and `load` can be used.
 docker save <image on host local repository> | docker --tlsverify --host tcp://$DEVICE_IP:$DOCKER_PORT load
 ```
 
+#### Using host user secondary groups in container
+
+The Docker Compose ACAP is run by a non-root user on the device. This user is set
+up to be a member in a number of secondary groups as listed in the
+[manifest.json](https://github.com/AxisCommunications/docker-compose-acap/blob/rootless-preview/app/manifest.json#L6-L11)
+file. When running a container a user called `root`, (uid 0), belonging to group `root`, (gid 0)
+will be the default user inside the container. It will be mapped to the non-root user on
+the device, and the group will be mapped to the non-root users primary group.
+In order to get access inside the container to resources on the device that are group owned by any
+of the non-root users secondary groups these need to be added for the container user.
```

Comment:
users'

Reply:
Perhaps a comman: "... by any of the non-root user's secondary groups, these need to.."

Reply:
The apostrophe is tricky
users' = possesion of several users
user's = user is
users = posesssion of one user

Reply:
Right, I was going for possession of several users, but maybe "user's" would have been better now that I look at it again. "user's" means possession of one user. This is a tiny detail in any case.